WordPress/src/main/res/values/strings.xml

```diff
@@ -1166,6 +1166,7 @@
     <string name="reader_empty_posts_request_failed">Unable to retrieve posts</string>
     <string name="reader_empty_posts_in_tag">No posts with this tag</string>
     <string name="reader_empty_posts_in_tag_updating">Fetching posts…</string>
+    <string name="reader_empty_posts_in_custom_list">The sites in this list haven\'t posted anything recently</string>
```

Comment:
I chose to use "haven't" rather than "have not" since that's more in line with [Google's writing style guidelines](https://www.google.com/design/spec/style/writing.html#writing-capitalization-punctuation).